jkube-kit/enricher/api/src/test/java/org/eclipse/jkube/kit/enricher/handler/ProbeHandlerHTTPUrlTest.java

```diff
@@ -15,40 +15,25 @@
 
 import io.fabric8.kubernetes.api.model.Probe;
 import org.eclipse.jkube.kit.config.resource.ProbeConfig;
-import org.junit.Before;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.junit.runners.Parameterized;
-
-import java.util.Arrays;
-import java.util.Collection;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullSource;
+import org.junit.jupiter.params.provider.ValueSource;
 
 import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
 
-
-@RunWith(Parameterized.class)
-public class ProbeHandlerHTTPUrlTest {
+class ProbeHandlerHTTPUrlTest {
   private ProbeHandler probeHandler;
 
-  @Parameterized.Parameters(name = "get HTTP Probe with {0} URL returns null probe")
-  public static Collection<Object[]> data() {
-    return Arrays.asList(
-        new Object[] { null},
-        new Object[] { "tcp://www.healthcheck.com:8080/healthz"},
-        new Object[] { "www.healthcheck.com:8080/healthz"}
-    );
-  }
-
-  @Parameterized.Parameter
-  public String url;
-
-  @Before
-  public void setUp() throws Exception {
+  @BeforeEach
+  void setUp() {
     probeHandler = new ProbeHandler();
   }
 
-  @Test
-  public void getProbe_withProvidedUrl_shouldGenerateHTTPProbe() {
+  @ParameterizedTest(name = "HTTP Probe with ''{0}'' URL returns null probe")
+  @NullSource
+  @ValueSource(strings = {"tcp://www.healthcheck.com:8080/healthz", "www.healthcheck.com:8080/healthz"})
```

Comment:
`null` test argument seems to be missing. You can use `@MethodSource` here

Reply:
According to [JUnit5 docs](https://javadoc.io/doc/org.junit.jupiter/junit-jupiter-params/5.8.2/org/junit/jupiter/params/provider/NullSource.html) `@NullSource` is an ArgumentsSource which provides a single null argument to the annotated `@ParameterizedTest` method.

Reply:
Sorry, I missed this annotation. Learned something new today :-)